bv-evaluation/collect-data-alive.py

```diff
@@ -4,7 +4,7 @@
 import os 
 import re
 
-paper_directory = 'for-paper/'
+paper_directory = ''
```

Comment:
@luisacicolini can you remind me why we removed this `for_paper/`? I think this change broke the CI

Reply:
i dont remember exactly, happy to take a look